package.json

```diff
@@ -4,9 +4,10 @@
   "description": "This repository was made for beginners to start learning Javascript from Scratch",
   "main": "index.js",
   "dependencies": {
+    "express": "^4.17.1",
+    "live-server": "^1.2.1",
```

Comment:
@Leannelb Just want to know for what task you were using live-server will love to know about it